pkg/deploymanager/subscription.go

```diff
@@ -97,7 +99,7 @@ func (t *DeployManager) generateClusterObjects(lvmCatalogImage string, subscript
 }
 
 // Waiting for LVM catalog source.
-func (t *DeployManager) waitForLVMCatalogSource() error {
+func (t *DeployManager) waitForLVMCatalogSource(catalogsource *v1alpha1.CatalogSource) error {
```

Comment:
It is enough to just pass the name of the catalogsource,

Reply:
Updated. Name is also not required now as we are following the previous method itself.